frontend/src/resources/policy.ts

```diff
@@ -55,8 +55,8 @@ export interface PolicyTemplate {
           subjects?: ResourceRef[]
         }
       }[]
-      remediationAction: string
-      severity: string
+      remediationAction?: string
```

Comment:
Current linting error:
```
error TS2322: Type 'string | undefined' is not assignable to type 'string'.
Type 'undefined' is not assignable to type 'string'. 
```
It looks like having `remediationAction` optional is causing this assignment to be upset?
https://github.com/stolostron/console/blob/9f5340910630f9541ed0ae69c90c29c0dfa18f98/frontend/src/routes/Governance/common/util.tsx#L484